src/spread/obj-overrides-prev-properties.case

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Object Spread properties overrides previous definitions
+template: default
+esid: pending
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- setup
+
+let o = {a: 2, b: 3}
+
+//- args
+{a: 1, b: 7, ...o}
+//- params
+obj
+//- body
+assert.sameValue(obj.a, 2);
+assert.sameValue(obj.b, 3);
+assert.sameValue(Object.keys(obj).length, 2);
```

Comment:
to make sure the values are assigned and not a reference to `o`, assert `o.a` and `o.b` are still `2` and `3`.